schema/products/eu-hydro.json

```diff
@@ -0,0 +1,168 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://example.com/vpp.json",
+  "title": "EU-Hydro River Network Database STAC Schema",
+  "description": "This schema only checks the existence but the schema of the properties",
+  "oneOf": [
```

Comment:
Is it necessary oneOf field? Cant be the fields directly without being inside oneof? just a question, i dont have context.

Reply:
No, it does not need to have oneOf there. I kept the oneOf just for the consistency, because the stac schema, e.g., the schema of extensions, always put item schema and collection schema together. 

What do you think?

Reply:
ok